spec/features/welcome/index_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'welcome page', type: :feature do
+
+  describe 'page appearance' do
+    it 'welcomes' do
+      visit '/'
+
+      expect(page).to have_content('Ingredient Search')
+    end
+  end
+
+  describe "Search for Ingredient" do
+    it "sends search to food show page" do
+      visit root_path
+
+      within ".form-inline" do
+        fill_in :q, with: 'sweet potatos'
+      end
+
+      click_on "Search"
+
+      expect(current_path).to eq('/foods')
```

Comment:
Would love to see that the total results and the correct number (10) of food objects and their attributes show up on this page.